src/main/java/core/basesyntax/CalculatorOfArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface CalculatorOfArea extends Drawable {
```

Comment:
```suggestion
public interface CalculatorOfArea {
```
you should implement Drawable in Figure

Reply:
Drawable implements in Figure now